apps/juicebox_web/test/lib/web/channels/stream_channel_test.exs

```diff
@@ -76,4 +76,56 @@ defmodule JuiceboxWeb.StreamChannelTest do
       assert_broadcast "remote.action", payload
     end
   end
+
+  describe "Connecting" do
+    setup [:with_some_reactions, :with_some_videos, :connect]
+
+    #test "pushes all current reactions", %{socket: socket} do
+    #  assert_push("remote.action", %{ video: "video 1", user_id: "user 1", "type": "NEW_REACTION" })
+    #  assert_push("remote.action", %{ video: "video 2", user_id: "user 2", "type": "NEW_REACTION" })
+    #  assert_push("remote.action", %{ video: "video 3", user_id: "user 3", "type": "NEW_REACTION" })
+    #end
+
+    test "pushes the current queue" do
+      assert_push("remote.action", %{ videos: [%{video: @video}, %{video: @video_2}], "type": "QUEUE_UPDATED" })
+    end
+  end
+
+  describe "Videos" do
+    setup [:connect, :receive_video]
+
+    test "video.added (in): adds a video to the stream" do
+      assert Stream.playing(@stream_id) == {:ok, %{video: @video}}
+    end
+  end
+
+  describe "Receiving a reaction" do
+    setup [:connect, :receive_reaction]
+
+    test "adds a reaction to the stream", %{id: id} do
+      assert Reactions.get(@stream_id, id) == "BASE64 VIDEO"
+    end
+
+    test "broadcasts the new reaction to all clients",
+          %{socket: socket, reaction_payload: payload, id: id} do
+
+      assert_broadcast("remote.action", %{ video: payload, user_id: id })
+    end
+  end
+
+  describe "Disconnecting" do
+    setup [:connect, :receive_reaction, :disconnect]
+
+    test "removes the user's reaction video", %{socket: socket} do
+      video = Reactions.get(@stream_id, socket.assigns.user_id)
+      assert video == nil
+    end
+
+    test "tells all clients to remove the reaction by broadcasting an empty reaction video",
+          %{socket: socket} do
+
+      assert_broadcast("remote.action", %{ video: nil, user_id: id })
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.